framework/src/mfem/bcs/MFEMConvectiveHeatFluxBC.C

```diff
@@ -26,13 +26,13 @@ MFEMConvectiveHeatFluxBC::validParams()
   params.addRequiredParam<MFEMScalarCoefficientName>(
       "T_infinity",
       "Name of a coefficient specifying the far-field temperature. A coefficient can be any of the "
-      "following: "
-      "a variable, an MFEM material property, a function, or a post-processor.");
+      "following: a variable, an MFEM material property, a function, a post-processor, or a "
+      "numeric value.");
```

Comment:
so for functors we made this explanation to be default for all MooseFunctorName. Should do the same here!